osmosis-1/osmosis-frontier.assetlist.json

```diff
@@ -1517,6 +1517,36 @@
       },
       "coingecko_id": "dai"
     },
+    {
+      "description": "Gravity Bridge BTC",
+      "denom_units": [
+        {
+          "denom": "ibc/C9B0D48FD2C5B91135F118FF2484551888966590D7BDC20F6A87308DBA670796",
+          "exponent": 0,
+          "aliases": [
+            "satoshi"
+          ]
+        },
+        {
+          "denom": "btc",
+          "exponent": 8
+        }
+      ],
+      "base": "ibc/C9B0D48FD2C5B91135F118FF2484551888966590D7BDC20F6A87308DBA670796",
+      "name": "Wrapped Bitcoin",
+      "display": "btc",
+      "symbol": "gWBTC",
+      "ibc": {
+        "source_channel": "channel-10",
+        "dst_channel": "channel-144",
+        "source_denom": "gravity0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/gwbtc.png",
+        "svg": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/gwbtc.svg"
```

Comment:
for now don't need this svg

Reply:
Removed the svg entry in logo_URIs and the file in images/